src/components/tl/mlx5/mcast/tl_mlx5_mcast.h

```diff
@@ -302,6 +303,9 @@ typedef struct ucc_tl_mlx5_mcast_bcast_comm {
     int           nacks_counter;
     int           n_mcast_reliable;
     int           wsize;
+    uint32_t      mcast_prepost_bucket_size;
+    uint8_t       truly_zero_copy_bcast_enabled;
+    int           coll_counter;
```

Comment:
just question why in some places bitness is specified like uint32_t and in others just int? Could counter be negative?

Reply:
To capture the wrong inputs from users (like negative ones), we are reading into an int